### ISSUE TITLE
CheckSession.html always return changed value

### ISSUE THREAD

**Issue by tejashdesai** [IdentityServer/IdentityServer3#2827]
Hi,

I am checking session by check_session_iframe. But it always return me change status. Is there any other way to check session on IdSrv?

Thanks,
Tejash

**Comment by tejashdesai** [comment]
Hi,

I have chacked that "clientHash" and "expectedHash" is different. I do not know why this is always different.

Thanks,
Tejash

**Comment by brockallen** [comment]
If the sid changed, then something has changed.

**Comment by tejashdesai** [comment]
Thanks for the reply. I checked sid and it is same as value stored into the idsvr.session cookie.

**Comment by brockallen** [comment]
Does the JS token manager sample work correctly?

**Comment by tejashdesai** [comment]
Yes. It is working correctly. But in my application "mgr.session_state" is always undefined. If i passed this explicitly then checksession.html always return change. I am not setting authorization using JS. I just want to use JS for checking session.

**Comment by brockallen** [comment]
Are you making OIDC requests, or just OAuth2 requests?

**Comment by tejashdesai** [comment]
Yes i am making OIDC request.

**Comment by tejashdesai** [comment]
Hi, Any update on this.

**Comment by brockallen** [comment]
Not sure -- what else have you investigated?
